code/SecureFileExtension.php

```diff
@@ -99,6 +90,35 @@ public function canView($member = null) {
 			return $file->Parent()->canView($member);
 		}
 
+		return $this->defaultPermissions($member);
+	}
+
+	/**
+	 * Checks for any default access permissions and tests against them if found. Default permssions are set via the 
+	 * Config system.
+	 *
+	 * @param Member $member
+	 * @return boolean
+	 */
+	protected function defaultPermissions($member = null) {
```

Comment:
Should this have an easy way of being extended or overridden?
Personally I feel there should be some ability to modify this maybe by adding a call for `$this->owner->extendDefaultPermmsions()`?